make-version/make-version.py

```diff
@@ -14,6 +14,20 @@ def update_json(version, pathname):
         fp.truncate()
         json.dump(data, fp, indent=2)
 
+
+# UPDATES VERSION ONLY IN TOP-LEVEL INFO SECTION
+def update_yaml(version, pathname):
```

Comment:
Is this really safe? Wouldn't it be a bit easier to install something like PyYAML, parse the YAML and change it and a more safe fashion?

Reply:
I wanted to avoid installing another packages and having whole file in memory

Reply:
Well, as long as it works safely, I am OK with it. It just looks a bit dangerous :)

Reply:
I'm pretty confident about this. Besides, this yaml is also validated in tests, so if it breaks something, we won't be able to make a release from it.